packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -260,14 +260,14 @@ class DatePickerDialog extends StatefulWidget {
     this.keyboardType,
     this.restorationId,
   }) : assert(initialDate != null),
-       assert(firstDate != null),
-       assert(lastDate != null),
-       initialDate = DateUtils.dateOnly(initialDate),
-       firstDate = DateUtils.dateOnly(firstDate),
-       lastDate = DateUtils.dateOnly(lastDate),
-       currentDate = DateUtils.dateOnly(currentDate ?? DateTime.now()),
-       assert(initialEntryMode != null),
-       assert(initialCalendarMode != null) {
+        assert(firstDate != null),
```

Comment:
This was not part of the diff in VS code. I have already redone the formatting 5 times because of auto save in this 4k line file. What is the way this is handled here ? Do I have to check the github diff to manually edit the formatting in the next commit ?

Reply:
I'm not sure if I have to remove those though, as prior to this change the formatting was on an odd column, this seems wrong